tests/functional/setupVpn.js

```diff
@@ -147,8 +159,17 @@ exports.mochaHooks = {
     }
     vpn.disconnect();
     vpnProcess.kill();
-    // Give each test 3 seconds to chill!
-    // Seems to help with tests that are slow to close vpn app at end.
-    await vpn.wait(3000);    
+
+    // Wait for process to close for ests that are slow to close vpn app at end.
+    let waitTimeForProcess = 0; // up to 10 sec wait
+    while (waitTimeForProcess < 11) {
```

Comment:
The process must be terminated. I would remove this while-stmt

Reply:
i thought about this, the waitTimeForProcess is there because i didnt want this to take 30-45 mins to timeout, from my research, if the process doesn't exit within that 10 secs, it wont ever exit